corefxlab-base.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
           - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
             - name: _buildArguments
-              value: -Configuration $(_BuildConfig)
+              value: -Configuration $(_BuildConfig) -test -ci
```

Comment:
add `-pack` here as well so that we exercise building the packages on PR builds as well.